README_EN.md

```diff
@@ -208,10 +208,9 @@ Check out class `PartialRexxarViewController` in RexxarDemo for example.
   - `NSDictionary+RXRMultipleItem`
 
 
-## Optimisation in future
-
-Replacing UIWebView by WKWebView is our long-term goal. WKWebView is better on speed and memory than UIWebView. But WKWebView doesn't support all features of UIWebView. The most import absence for Rexxar is the support of NSURLProtocol. We can not intercept the request from Rexxar Container with WKWebView. Anyway we will do our best to bring WKWebView to Rexxar in future.
+## Changelog
 
+- 0.3.0   Replace UIWebView with WKWebView. But WKWebView support for NSURLProtocol is not good. The body of the requests which http method is POST will be clean when the requests been interrupted by NSURLProtocols. So you need to take care of that.
 
```

Comment:
But WKWebView support for NSURLProtocol is not good
-> NSURLProtocol is not supported in WKWebView.

The body of the requests which http method is POST will be clean when the requests been interrupted by NSURLProtocols. 
-> The body of the POST request(HTTP) will be cleaned when intercepted by NSURLProtocol.